frontend/app/dashboard/products/page.tsx

```
@@ -0,0 +1,1490 @@
+                            aria-label="Upload product images"
+                          />
+                        </label>
+                      </div>
+
+                      {imageUrls.length > 0 && (
+                        <div className="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-4 mt-4">
+                          {imageUrls.map((url, index) => (
+                            <div key={index} className="relative group">
+                              <img
+                                src={url || "/placeholder.svg"}
+                                alt={`Product image ${index + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)